AI-and-Analytics/Getting-Started-Samples/INC-Sample-for-Tensorflow/README.md

```diff
@@ -72,124 +73,170 @@ The sample demonstrates how to:
 
 ```
-conda info -e
-# conda environments:
-#
-base                  *  /opt/intel/oneapi/intelpython/latest
-pytorch                  /opt/intel/oneapi/intelpython/latest/envs/pytorch
-pytorch-1.7.0            /opt/intel/oneapi/intelpython/latest/envs/pytorch-1.7.0
-tensorflow               /opt/intel/oneapi/intelpython/latest/envs/tensorflow
-tensorflow-2.3.0         /opt/intel/oneapi/intelpython/latest/envs/tensorflow-2.3.0
-                         /opt/intel/oneapi/pytorch/1.7.0
-                         /opt/intel/oneapi/tensorflow/2.3.0
+source $HOME/intel/oneapi/intelpython/bin/activate
 ```
-1. Activate the conda environment with Intel® Optimizations for TensorFlow*.
-
-   By default, the Intel® AI Analytics Toolkit is installed in
-   the `/opt/intel/oneapi` folder, which requires root privileges to manage it.
-
-   1. If you have the root access to your oneAPI installation path:
-       ```
-       conda activate tensorflow
-       (tensorflow) xxx@yyy:
-       ```
-
-   2. If you do not have the root access to your oneAPI installation path, clone the `tensorflow` conda environment using the following command:
-      ```
-      conda create --name usr_tensorflow --clone tensorflow
-       ```
-
-   3. Activate your conda environment with the following command:
-      ```
-      source activate usr_tensorflow
-      ```
-2. Install Intel® Neural Compressor from the local channel.
-   ```
-   conda install -c ${ONEAPI_ROOT}/conda_channel neural-compressor -y --offline
-   ```
 
-##### Option 2: Create Conda Environment
+If a non-default path is used:
 
-Configure Conda for **user_tensorflow** by entering commands similar to the following:
-   ```
-   conda deactivate
-   conda env remove -n user_tensorflow
-   conda create -n user_tensorflow python=3.9 -y
-   conda activate user_tensorflow
-   conda install -n user_tensorflow pycocotools -c esri -y
-   conda install -n user_tensorflow neural-compressor tensorflow -c conda-forge -c intel -y
-   conda install -n user_tensorflow jupyter runipy notebook -y
-   conda install -c anaconda ipykernel
-   python -m ipykernel install --user --nam=user_tensorflow
-   ```
+```
+source <custom_path>/bin/activate
+```
 
+### 3. (Offline Installer) Activate relevant Conda environment
 
-#### Register Conda kernel to Jupyter Notebook kernel
+```
+conda activate tensorflow
+```
 
-Create a new kernel for the Jupyter notebook based on your activated conda environment.
-   ```
-   conda install ipykernel
-   python -m ipykernel install --user --name usr_tensorflow
-   ```
-   This step is optional if you plan to open the notebook on your local server.
+### 4. Clone the GitHub repository
+
+```
+git clone https://github.com/oneapi-src/oneAPI-samples.git
+cd oneAPI-samples/AI-and-Analytics/Getting-Started-Samples/INC-Sample-for-Tensorflow
+```
+
+### 5. Install dependencies
+
+Install for Jupyter Notebook:
+
+```
+conda install jupyter runipy notebook -y
```

Comment:
it's recommended to but dependent packages into requirements.txt file and install everything with `pip install -r requirements.txt`.
In any case, we need to make sure we list all the needed packages, right now I see that ipykernel and matplotlib packages are missing in the command

Reply:
yes, create requirements.txt